lib/MTT/Test/RunEngine.pm

```diff
@@ -222,7 +222,7 @@ sub RunEngine {
         $run->{test_build_simple_section_name} = $test_build->{simple_section_name};
 
         # Setup some globals
-        $MTT::Test::Run::test_build_dir = $test_build->{srcdir};
+        $MTT::Test::Run::test_build_dir = $test_build->{bindir};
```

Comment:
This might be a similar case as before: the *build* directory is different than the *source* or the *install* directory.  This variable is returned by the funclet `test_build_dir()`, but note that we already have a `test_executable_dir()`.

The build directory is *usually* the source directory, but it could also be a VPATH build tree.

But I don't think it's correct to default it to the bin directory.

Reply:
OK, need to think. The previous chunk+comment are related to this one, most likely will be changed together.

Reply:
Cool.  Thanks!